scripts/contacts/migrateContactsV1toV2.spec.js

```diff
@@ -155,10 +155,8 @@ describe('Contacts: migrate cozy metadata (konnector v1 to v2)', () => {
         email: [],
         address: [],
         metadata: {
-          version: 1,
           google: { metadata: {}, from: 'appvengers.cedric@gmail.com' }
```

Comment:
Not sure about this — how do we store the google metadata in the konnector? Directly in `metadata` or in `metadata.google`?

Reply:
in the transpiler it looks like this (and my fixture comes from real data):

```      
metadata: {
  version: SCHEMA_VERSION,
  google: { metadata: source.metadata }
},
```